mindbender/plugins/collect_project_editorial_info.py

```diff
@@ -0,0 +1,36 @@
+# collect_project_editorial_info.py
+import pyblish.api
+
+
+class CollectMindbenderEditInfo(pyblish.api.ContextPlugin):
+    """Store projects editorial info at the time of publish"""
+
+    label = "Collect Project Edit Info"
+    order = pyblish.api.CollectorOrder
+    hosts = ["maya"]
+    families = [
+        "mindbender.animation"
+    ]
+
+    def process(self, context):
+        import os
+
+        if "environment" not in context.data:
+            context.data["environment"] = dict()
+
+        prefix = "MINDBENDER_"
+        for key, value in os.environ.items():
+            if not key.startswith(prefix):
+                continue
+
+            key = key[len(prefix):]
+
+            # Convert EDIT_IN to EditIn, FPS to Fps.
+            key = "".join(k.title() for k in key.split("_"))
+
+            # Decorate key with "mindbender" to match prev namning convention
+            decoration = prefix.replace("_", "")
```

Comment:
Nice, but we can achieve the same result by removing line 26.

```python
key = key[len(prefix):]
```

Let me tell you how it works.

- `len(prefix)` returns the length of the string `prefix`, which is `"MINDBENDER_"`, which is 11 characters long.
- `key[11:]` says "Give me the content of `key`, starting 11 characters in and continuing till the end, `:`. That's because strings are treated like a list, where each character is a member of this list. For example, `[1, 2, 3][0]` returns `1`, and `"123"[0]` returns `"1"`. Using the `:` operator is called "slicing". We are *slicing* a list starting from some index, such as 11, to some other index. E.g. `[1, 2, 3][0:2]` would give us `[1, 2]`. If you don't specify an end, it will assume you meant the last index. `[1, 2, 3][0:] == [1, 2, 3]`. And that's how you slice. `[1, 2, 3][1:] == [2, 3]` and `"MINDBENDER_FPS"[11:] == "FPS"`

In this case, I purposefully discarded `MINDBENDER_` because this is what we use to namespace variables in the environment. It doesn't actually add any value once read, so we don't really need it.

Let me know if this makes sense.